internal/server/pages/index.html

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
```

Comment:
Just noticed this was missing because the browser inspector was complaining about it.